File: src/main/java/szewek/mcflux/blocks/itemblocks/ItemMCFluxBlock.java
package szewek.mcflux.blocks.itemblocks;

import net.minecraft.block.Block;
import net.minecraft.client.resources.I18n;
import net.minecraft.client.util.ITooltipFlag;
import net.minecraft.item.ItemBlock;
import net.minecraft.item.ItemStack;
import net.minecraft.world.World;

import javax.annotation.Nullable;
import java.util.List;

public final class ItemMCFluxBlock extends ItemBlock {
	public ItemMCFluxBlock(Block block) {
		super(block);
	}

	@Override
	public void addInformation(ItemStack stack, @Nullable World worldIn, List<String> tooltip, ITooltipFlag flagIn) {
		tooltip.add(I18n.format(getUnlocalizedName(stack) + ".desc"));
		super.addInformation(stack, worldIn, tooltip, flagIn);
	}
}

File: src/main/java/szewek/mcflux/blocks/itemblocks/ItemMCFluxBlock.kt
package szewek.mcflux.blocks.itemblocks

import net.minecraft.block.Block
import net.minecraft.client.resources.I18n
import net.minecraft.client.util.ITooltipFlag
import net.minecraft.item.ItemBlock
import net.minecraft.item.ItemStack
import net.minecraft.world.World

class ItemMCFluxBlock(block: Block) : ItemBlock(block) {

	override fun addInformation(stack: ItemStack, worldIn: World?, tooltip: MutableList<String>, flagIn: ITooltipFlag) {
		tooltip += I18n.format(getUnlocalizedName(stack) + ".desc")
		super.addInformation(stack, worldIn, tooltip, flagIn)
	}
}
